mobile/lib/src/application/blocs.dart

```diff
@@ -0,0 +1,3 @@
+export './device/device_selector_cubit.dart';
+export './eeg_data/data_cubit.dart';
+export './navdrawer/navdrawer_bloc.dart';
```

Comment:
We decided not to export packages as a whole. Does this also include blocs? @WilliamHarvey97 @MouradLachhab

Reply:
In that case, I think it's ok like that. The issue was more about exporting the whole layer.